dsmr_parser/clients/serial_.py

```diff
@@ -50,15 +56,20 @@ def read_as_object(self):
         with serial.Serial(**self.serial_settings) as serial_handle:
             while True:
                 data = serial_handle.readline()
-                self.telegram_buffer.append(data.decode('ascii'))
+                try:
+                    decoded_data = data.decode('ascii')
+                except:
+                    pass
```

Comment:
Maybe put a logger statement here:
`logger.warning('Failed to decode telegram data: %s', e)`